templates/navigation/_dropdown.html

```diff
@@ -2,7 +2,7 @@
      data-level="1"
      id="{{ id }}-content"
      aria-hidden="true">
-  <div class="p-navigation__dropdown-content--sliding">
+  <div class="p-navigation__dropdown-content-mobile">
```

Comment:
Why was this changed from sliding to mobile? It's currently showing the mobile dropdown on desktop view

Reply:
There were some changes in vanilla I was trying to keep up with (as I am working off of main). I have reverted this now and it should be fixed